ceagle/api/v1/status.py

```diff
@@ -13,15 +13,34 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
+import json
+
 import flask
+import requests
```

Comment:
No need to use requests directly, there is an API client for that
https://github.com/seecloud/ceagle/blob/c749cf1d1a148ed4ca12ed409febea42b48b71c8/ceagle/api/client.py#L56